### DISCUSSION TITLE
StudioElf.Template.Syncfusion 27.1.56 released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4771] [Show and tell]
StudioElf.Template.Syncfusion 27.1.56 released
Available in the marketplace
![image](https://github.com/user-attachments/assets/33d6a45a-fccc-4412-bd74-faa440a58f07)

**Comment by LuisVilaplana** [comment]
Hi, I was trying to use your templates but I can't get the .nupkg file to be created, which causes the new module not to appear in Oqtane. Any workaround?

**Reply by leigh-pointer** [reply]
@LuisVilaplana uninstall the template 27.1.56.  Double Check the removal in the ;

```
Oqtane.Framework\Oqtane.Server\wwwroot\Modules\Templates\
Oqtane.Framework\Oqtane.Server\wwwroot\Themes\Templates\
```

and install version 28.1.39
